src/control/shotControllers/ShotController.cpp

```diff
@@ -164,7 +164,7 @@ RobotCommand ShotController::moveStraightToBall(world::Robot robot,
     //auto newPid = interface::Output::getNumTreePid();
     auto newPidValues = interface::Output::getShotControllerPID();
     updatePid(newPidValues);
-    double pidOutput = pid.getOutput(err.length()*sign, 0);
+    double pidOutput = pid.computeOutput(err.length() * sign, 0);
```

Comment:
Indeed there should be a space between the multiplication operator and after the multiplication operator :+1: